packages/terra-abstract-modal/src/terra-dev-site/test/abstract-modal/AbstractModalTestStyles.scss

```diff
@@ -0,0 +1,11 @@
+.overlay-custom-class {
```

Comment:
Is there anything preventing us from doing CSS modules here? It doesn't add a lot of complexity, and it ensures we have styles that will not collide.